tasks/hidden_service.yml

```diff
@@ -4,7 +4,7 @@
 
 - name: Add the HiddenServiceDir config to torrc
   blockinfile:
-    dest: /etc/tor/torrc
+    dest: "{{ torrc }}"
```

Comment:
that's kinda the whole problem of a scripting language due to being weakly typed, you can only catch errors at runtime. That's not great and I agree with it. Solution usually is to have complete tests, so that can be fixed, but then testing all code path is expensive and long :/